src/components/Filters/Filters.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const Filters = ({ setFilterStatus, filterStatus, activeSelectAll }) => (
+  <ul className="filters">
+    <li>
+      <button
+        type="button"
+        className={classNames('filter', {
+          selected: filterStatus === 'all',
+        })}
+        onClick={() => setFilterStatus('all')}
```

Comment:
create a constant with all filters names, so you can use it like this  -> FILTERS.all, etc
(save it in a separate file)